internal/server/v0/placestatvar/golden/get_place_stat_vars/california.json

```diff
@@ -34733,7 +34743,6 @@
         "dc/03cf1l5hv9j73",
         "dc/03eg4wmfv5t24",
         "dc/03eqbgjbqvtlc",
-        "dc/03ewbkyh6zey2",
```

Comment:
@shapateriya @manishvats2 Can you pls confirm if this deletion from USNationalPrisonerStatistics is expected in autorefresh. These Svs were present in the manual refresh cl/781014968